src/main/resources/jenkins/plugins/logstash/LogstashBuildWrapper/help-script.html

```diff
@@ -0,0 +1,37 @@
+<p>
+  With this script you can filter, modify and/or group messages that are sent to the
+  configured logstash backend. The following variables are available during execution:
+  <ul>
+    <li><strong>payload</strong> - JSONObject contaning the payload to be persisted.</li>
+    <li><strong>console</strong> - a class to output messages to build log without persisting them. It also works in a sandbox: <code>console.println("my logged message")</code>.</li>
```

Comment:
Since pipeline is using an `echo` statement maybe it would be nice to use the same (at least make it look the same). 
```
echo "my logged message"
```

Reply:
I don't know how to do this. I didn't find any way to add a method to the binding. And using `org.codehaus.groovy.runtime.MethodClosure` to create a closure property in the binding didn't work either with some script security issue (I tried this already). Maybe my groovy shell foo is lacking. Any ideas would be welcome.

Reply:
Why not bind `groovy.lang.Closure`? 

If that doesn't work then lets drop it, it's only nice to have

Reply:
will try

Reply:
FYI doing `echo "some string"` in script resulted in 
```
RejectedAccessException: Scripts not permitted to use method groovy.lang.GroovyObject invokeMethod java.lang.String java.lang.Object (Script1 echo java.lang.String)
```

While `echo.call "some string"` worked. I couldn't find a way to get around that.